--- makefile.in
+++ makefile.in
@@ -18,6 +18,7 @@
 S=-DWORDSIZE=16
 W=-DWORDSIZE=32
 L=-DWORDSIZE=64
+AR?=ar
 
 CCOBJ=${CC} -c ${CFLAGS} -o $@
 GTOOLSH=gtools.h nauty.h naututil.h nausparse.h naurng.h
@@ -207,38 +208,38 @@
 nauty.a: ${NAUTYO} traces.o gtools.o naututil.o nautinv.o \
 			gutil1.o gutil2.o gtnauty.o naugroup.o
 	rm -f nauty.a
-	ar crs nauty.a ${NAUTYO} traces.o gtools.o naututil.o \
+	${AR} crs nauty.a ${NAUTYO} traces.o gtools.o naututil.o \
 		 nautinv.o gutil1.o gutil2.o gtnauty.o naugroup.o \
 		 naurng.o schreier.o
 
 nauty1.a: ${NAUTY1O} traces.o gtools.o naututil1.o nautinv1.o \
 	   gutil1.o gutil2.o gtnauty.o naugroup.o
 	rm -f nauty1.a
-	ar crs nauty1.a ${NAUTY1O} traces.o gtools.o naututil1.o \
+	${AR} crs nauty1.a ${NAUTY1O} traces.o gtools.o naututil1.o \
 	   nautinv1.o gutil1.o gutil2.o gtnauty.o naugroup.o 
 
 nautyW.a: ${NAUTYWO} traces.o gtoolsW.o naututilW.o nautinvW.o \
 	   gutil1W.o gutil2W.o gtnautyW.o naugroupW.o 
 	rm -f nautyW.a
-	ar crs nautyW.a ${NAUTYWO} traces.o gtoolsW.o naututilW.o \
+	${AR} crs nautyW.a ${NAUTYWO} traces.o gtoolsW.o naututilW.o \
 	   nautinvW.o gutil1W.o gutil2W.o gtnautyW.o naugroupW.o
 
 nautyW1.a: ${NAUTYW1O} traces.o gtoolsW.o naututilW1.o nautinvW1.o \
 	   gutil1W1.o gutil2W1.o gtnautyW1.o naugroupW.o 
 	rm -f nautyW1.a
-	ar crs nautyW1.a ${NAUTYW1O} traces.o gtoolsW.o naututilW1.o \
+	${AR} crs nautyW1.a ${NAUTYW1O} traces.o gtoolsW.o naututilW1.o \
 	   nautinvW1.o gutil1W1.o gutil2W1.o gtnautyW1.o naugroupW.o
 
 nautyL.a: ${NAUTYLO} traces.o gtoolsL.o naututilL.o nautinvL.o \
 	   gutil1L.o gutil2L.o gtnautyL.o naugroupL.o 
 	rm -f nautyL.a
-	ar crs nautyL.a ${NAUTYLO} traces.o gtoolsL.o naututilL.o \
+	${AR} crs nautyL.a ${NAUTYLO} traces.o gtoolsL.o naututilL.o \
 	   nautinvL.o gutil1L.o gutil2L.o gtnautyL.o naugroupL.o
 
 nautyL1.a: ${NAUTYL1O} traces.o gtoolsL.o naututilL1.o nautinvL1.o \
 	   gutil1L1.o gutil2L1.o gtnautyL1.o naugroupL.o 
 	rm -f nautyL1.a
-	ar crs nautyL1.a ${NAUTYL1O} traces.o gtoolsL.o naututilL1.o \
+	${AR} crs nautyL1.a ${NAUTYL1O} traces.o gtoolsL.o naututilL1.o \
 	   nautinvL1.o gutil1L1.o gutil2L1.o gtnautyL1.o naugroupL.o
 
 clean:
