--- traces.c
+++ traces.c
@@ -300,8 +300,10 @@
 static int Preprocess_refine(sparsegraph*, permnode**, Candidate*, int, Partition*, struct TracesVars*);
 static void MakeTree(int, int, sparsegraph*, int, struct TracesVars*, boolean);
 static void MakeCanTree(int, sparsegraph*, int, Candidate*, Partition*, struct TracesVars*);
+#ifndef _MSC_VER
 static int max(int, int);
 static int min(int, int);
+#endif
 static void orbjoin_sp_perm(int*, int*, int*, int, int*);
 static void orbjoin_sp_pair(int*, int*, int, int, int, int*);
 static boolean isautom_sg_pair(graph*, int*, boolean, int, int, struct TracesVars*);
@@ -8686,6 +8688,7 @@
     return;
 }
 
+#ifndef _MSC_VER
 int max(int u, int v) {
     if (u > v) {
         return u;
@@ -8703,6 +8706,7 @@
         return v;
     }
 }
+#endif
 
 int NextNeighbour(int vtx, Candidate *Cand, Partition *Part, int* Markers, int mark, int *ngh, int n) {
     int *e_vtx;
